pkg/apis/certmanager/v1alpha1/types_issuer.go

```diff
@@ -182,6 +188,18 @@ type VaultAppRole struct {
 	SecretRef SecretKeySelector `json:"secretRef"`
 }
 
+type KubernetesAuth struct {
```

Comment:
Please add a comment to the struct too - it makes it a lot clearer when a user runs `kubectl explain` then 😄